src/patterns/components/links/react/info/inline.hbs

```diff
@@ -41,7 +41,7 @@
       </td>
 
       <td>
-        The URL of the link.
+        The URL of the link. Defaults to '#' and preventDefault() if no href is provided.
       </td>
     </tr>
 
```

Comment:
Double check that this list matches the component. `onClick`, `target`, `element`